### DISCUSSION TITLE
Redis like KV support for caching

### DISCUSSION THREAD

**Discussion by aarjan** [supabase/supabase#8659] [Questions] [answered]
Hi community, 

I am trying to implement a caching layer in my flutter app before hitting supabase db. 

Something like Redis store or, https://www.cloudflare.com/products/workers-kv/ 
would have been awesome.

Any advice on how could i achieve this would be great!

**Comment by dshukertjr** [comment, marked as answer]
Here is a free course on how to implement caching using Supabase and cloud flare workers that will most likely cover your use case!
https://egghead.io/courses/cache-supabase-data-at-the-edge-with-cloudflare-workers-and-kv-storage-883c7959

**Reply by aarjan** [reply]
Thank you very much, It's a good tutorial.
